### PR TITLE
feat: 1 dollar bet modal removed in checkout page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- $1 bet modal removed in checkout page

### CODE REVIEW

**Comment by dyersituations** [comment]
@roger-in-kiva Oh I think we wanted to wait to do this work until after IWD. Could you post in our team slack channel and tag Mona to verify? I'll wait to approve until we hear back.

**Comment by roger-in-kiva** [comment]
> @roger-in-kiva Oh I think we wanted to wait to do this work until after IWD. Could you post in our team slack channel and tag Mona to verify? I'll wait to approve until we hear back.

@dyersituations @mcstover we are ready to go with this.

**Comment by dyersituations** [comment]
@roger-in-kiva For testing purposes on dev, when does the `DonationNudgeLightbox.vue` get displayed now that the 1 dollar bet isn't being shown?

**Comment by roger-in-kiva** [comment]
> > > @roger-in-kiva For testing purposes on dev, when does the `DonationNudgeLightbox.vue` get displayed now that the 1 dollar bet isn't being shown?
> 
> > 
> 
> > We have it in the `DonationItem` component;
> 
> > 
> 
> > https://github.com/kiva/ui/blob/b0664dfd89da8e28441e3bc3c78117e9be72322d/src/components/Checkout/DonationItem.vue#L192
> 
> 
> 
> How do we get it to display?

When having a loan on checkout and want to edit the donation/tip.

File: src/components/Checkout/DonationNudge/DonationNudgeLightbox.vue [hunk 1]
```diff
@@ -69,38 +63,16 @@
...
-			</kv-button>
-		</div>
 	</kv-lightbox>
 </template>
 
 <script>
 import DonationNudgeBoxes from '@/components/Checkout/DonationNudge/DonationNudgeBoxes';
 import KvCharityNavigator from '@/components/Kv/KvCharityNavigator';
 import { mdiInformation } from '@mdi/js';
-import HeartIcon from '@/assets/icons/inline/heart-icon.svg';
```
Comment (dyersituations) [inline src/components/Checkout/DonationNudge/DonationNudgeLightbox.vue:98]: Looks like this asset file isn't being used anywhere else and can be removed.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**